modules/db_parameter_group/main.tf

```diff
@@ -1,8 +1,24 @@
+resource "aws_db_parameter_group" "this_no_prefix" {
```

Comment:
Update https://github.com/terraform-aws-modules/terraform-aws-rds/blob/master/modules/db_parameter_group/outputs.tf to include `aws_db_parameter_group.this_no_prefix` resource also.

Reply:
Done